soc/st/stm32/stm32h5x/power.c

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (c) 2024 STMicroelectronics.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include <zephyr/kernel.h>
+#include <zephyr/pm/pm.h>
+#include <soc.h>
+#include <zephyr/init.h>
+
+#include <stm32h5xx_ll_utils.h>
+#include <stm32h5xx_ll_bus.h>
+#include <stm32h5xx_ll_cortex.h>
+#include <stm32h5xx_ll_pwr.h>
+#include <stm32h5xx_ll_system.h>
```

Comment:
Are all these files actually required ?
At first sight, I would say that utils, bus and system could be removed ?